### PR TITLE
Update tables in proofreading tutorials to use a defined class

### PR DESCRIPTION
**Pull request by srjfoo**
Table in the proofreading tutorials were not themed. Set them all to the "basic" table style.

This primarily just adds a table class and cleans out unnecessary attributes, and tries to verify that the alignment matches. I've taken out some alignment attributes where they didn't seem to be doing anything. I did not try to change the basic structure of the code that's there, but can do so if desirable (with a recommended pattern to follow).

This changes the `<th>` colors to use the basic table heading colors; that will probably be the most noticeable difference except for a couple of the tables that were set to 100%. I did not put that back on, because it seems to me to be unnecessary.

I left the summaries in place on the tables that already had them -- should I add them for those that don't have them?

Test in the [update-tuts-tables](https://www.pgdp.org/~srjfoo/c.branch/update-tuts-tables/quiz/start.php?show_only=proof) sandbox. Compare to the [current version](https://www.pgdp.org/c/quiz/start.php?show_only=proof).

### CODE REVIEW

**Review by chrismiceli — APPROVED** [review]

File: quiz/tuts/tut_p_mod1_1.php [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
 echo "<p>" . _("Older books often abbreviated words as contractions, and printed them as superscripts. Proofread these by inserting a single caret (<kbd>^</kbd>) followed by the superscripted text. If the superscript continues for more than one character, then surround the text with curly braces <kbd>{</kbd> and <kbd>}</kbd> as well. For example:") . "</p>\n";
-echo "<table width='100%' align='center' border='1' cellpadding='4' cellspacing='0' summary='" . _("Superscripts example") . "'>\n";
-echo "  <tbody>\n";
-echo "    <tr><th align='left' bgcolor='cornsilk'>" . _("Original Image:") . "</th></tr>\n";
+echo "<table class='basic' summary='" . _("Superscripts example") . "'>\n";
+echo "  <tbody class='left-align'>\n";
+echo "    <tr><th>" . _("Original Image:") . "</th></tr>\n";
 echo "    <tr>\n";
-echo "      <td valign='top'>Gen<sup>rl</sup> Washington defeated L<sup>d</sup> Cornwall's army.</td>\n";
+echo "      <td>Gen<sup>rl</sup> Washington defeated L<sup>d</sup> Cornwallis's army.</td>\n";
```
Comment (cpeel) [inline quiz/tuts/tut_p_mod1_1.php:21]: Was the change from `Cornwall's` to `Cornwallis's` intentional?

**Review by cpeel — COMMENTED** [review]
What you have looks good to me. One question about a non-style change but I'm great otherwise.

File: quiz/tuts/tut_p_mod1_1.php (hunk 1 above)
Comment (srjfoo) [reply quiz/tuts/tut_p_mod1_1.php:21]: Yes -- the corresponding mention in the Proofreading Guidelines was changed some time back. I verified with @lhamilton1 before I made the change here.

Comment (cpeel) [reply quiz/tuts/tut_p_mod1_1.php:21]: Great - thanks!

**Pull request merged**